packages/vue-components/src/TipBox.vue

```diff
@@ -1,92 +1,108 @@
 <template>
-  <div :class="['alert box-container', containerStyle(), addClass]" :style="customStyle()">
-    <!-- Header wrapper, not rendered if there is no header attribute -->
-    <div v-if="headerBool()" :class="['box-header-wrapper', { 'alert-dismissible': dismissible }]">
+  <div
```

Comment:
we try to limit PRs, and especially commits to one logical change to make things easy to trace.

let's defer the style changes / fixes (and for the below `<style>`) to another PR. (the stylelint configurations also need some fixing, I think)

Reply:
Hi I'm not sure what style changes should be deferred here. I have restored the original code and made minimal changes to achieve desired UI. Please let me know what I should change further, thanks